utils/firestore.js

```diff
@@ -2375,6 +2400,7 @@ const confirmShipmentKit = async (shipmentData) => {
         const prefEmail = participantDocData['869588347'];
         const token = participantDocData['token'];
         const ptName = participantDocData['153211406'] || participantDocData['399159511']
+        const preferredLanguage = cidToLangMapper[participantDocData[fieldMapping.preferredLanguage]] || "english";
```

Comment:
NIT: Should we define `english` in `shared.js` . We can reuse it somewhere.

Reply:
Not needed, becuase `english` is defined in `cidToLangMapper` and `fieldMapping` already.
A version without "english" can be: 
```js
const preferredLanguage = cidToLangMapper[participantDocData[fieldMapping.preferredLanguage]] || cidToLangMapper[fieldMapping.english]
```